packages/home-spa/.env.example

```diff
@@ -6,3 +6,4 @@ OP_SUBSCRIPTIONS_URL=
 
 ## SENTRY DSN
 # SENTRY_DSN=
+# Checking changed files
```

Comment:
Can you please remoce the duplicate env variable and comment

Reply:
@riginoommen both are required in home-spa

Reply:
One is the actual variable and other is the comment